File: TransCoder_tokenized_test_set_functions/python/FIND_THE_TWO_REPEATING_ELEMENTS_IN_A_GIVEN_ARRAY_3.py
def printRepeating ( arr, size ) :
    print ( " The repeating elements are", end = " " )
    for i in range ( 0, size ) :
        if ( arr [ abs ( arr [ i ] ) ] > 0 ) :
            arr [ abs ( arr [ i ] ) ] = ( - 1 ) * arr [ abs ( arr [ i ] ) ]
        else :
            print ( abs ( arr [ i ] ), end = " " )
    

File: TransCoder_tokenized_test_set_functions/java/FIND_THE_TWO_REPEATING_ELEMENTS_IN_A_GIVEN_ARRAY_3.java
void printRepeating ( int arr [ ], int size ) {
  int i;
  System . out . println ( "The repeating elements are : " );
  for ( i = 0;
  i < size;
  i ++ ) {
    if ( arr [ Math . abs ( arr [ i ] ) ] > 0 ) arr [ Math . abs ( arr [ i ] ) ] = - arr [ Math . abs ( arr [ i ] ) ];
    else System . out . print ( Math . abs ( arr [ i ] ) + " " );
  }
}